src/test/java/com/hazelcast/kubernetes/ApplicationTest.java

```diff
@@ -0,0 +1,44 @@
+package com.hazelcast.kubernetes;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.config.ClientNetworkConfig;
+import com.hazelcast.config.KubernetesConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+class ApplicationTest {
+
+
+    @Test
+    void shouldConnectToHazelcastCluster() {
+        String masterIp = System.getenv("KUBERNETES_MASTER").replaceAll("\u001B\\[[;\\d]*m", "");
+        ClientConfig config = new ClientConfig();
+        config.setInstanceName("dev");
+        KubernetesConfig kubernetesConfig = new KubernetesConfig();
+        kubernetesConfig.setEnabled(true);
+        kubernetesConfig.setProperty("kubernetes-master", masterIp);
+        kubernetesConfig.setProperty("namespace", "default");
+        kubernetesConfig.setProperty("api-token", System.getenv("API_TOKEN"));
+        kubernetesConfig.setProperty("ca-certificate", System.getenv("CA_CERTIFICATE"));
+        kubernetesConfig.setUsePublicIp(true);
+        ClientNetworkConfig networkConfig = new ClientNetworkConfig().setKubernetesConfig(kubernetesConfig);
+        config.setNetworkConfig(networkConfig);
+
+        HazelcastInstance hazelcastInstance = HazelcastClient.newHazelcastClient(config);
+        IMap<String, String> mapToPut = hazelcastInstance.getMap("map");
+        mapToPut.put("key", "value");
+
+        IMap<Object, Object> mapToGet = hazelcastInstance.getMap("map");
```

Comment:
Does this `getMap` have a particular purpose? Otherwise, I believe it's not necessary.

Reply:
agree, will remove it